package.json

```diff
@@ -41,16 +44,22 @@
     ]
   },
   "dependencies": {
+    "@shopgate/cloud-sdk-webpack": "github:shopgate/cloud-sdk-webpack",
```

Comment:
if you want to use github instead of npm please add a tag, otherwise this is a very unreliable dependency
alternativly push cloud-sdk-webpack to npm